torchelie/nn/layers.py

```diff
@@ -92,7 +92,6 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         v = self.value(x_flat)
 
         affinity = torch.einsum('bki,bkj->bij', q, k)
-        attention = F.softmax(affinity, dim=1)
```

Comment:
This is a bug indeed. The einsum() below should use attention rather than affinity

Reply:
Opened issue #10 to make sure to not forget about it! I guess it might make sense to remove the dead variable in the meantime since the issue have a link to a commit that still has the line?

Reply:
Solved by 02a9f2f73848e48381e3b824f62c7006b3a9eab2